src/pravin/Assignment_14/IsVowelWithScanner.java

```diff
@@ -0,0 +1,44 @@
+package pravin.Assignment_14;
+
+	import java.util.Scanner;
+
+
+	public class IsVowelWithScanner {
```

Comment:
code logic right, instead of writing logic in main method you should write in separate method